scripts/normalize_jest_coverage.js

```diff
@@ -0,0 +1,27 @@
+const libCoverage = require("istanbul-lib-coverage")
```

Comment:
Can this file be made `camelCase` -- `normalizeJestCoverage.js`? Snake-case has typically been deprecated for new code.